src/services/kpis/kpis.svc.coffee

```diff
@@ -42,10 +56,12 @@ angular
     @load = (force=false) ->
       deferred = $q.defer()
 
-      if _.isEmpty(_self.config.ssoSessionId) || force
+      if _.isEmpty(_self.getSsoSessionId()) || force
         ImpacMainSvc.loadUserData(force).then(
           (mainConfig) ->
-            _self.config.ssoSessionId = mainConfig.sso_session
+            _self.config.userData.id = mainConfig.id
+            _self.config.userData.orgUids = _.map(mainConfig.organizations, (o)-> o.uid)
+            _self.config.userData.ssoSessionId = mainConfig.sso_session if mainConfig.sso_session
```

Comment:
Do we really need this? If we have a `getSsoSessionId()` method that is used everywhere in this service, it could just return `MainSvc.config.sso_session` so that the sso session is stored at only one place...